packages/xrpl/src/sugar/getOrderbook.ts

```diff
@@ -40,41 +48,84 @@ function sortOffers(offers: BookOffer[]): BookOffer[] {
  * the order book. Defaults to 20.
  * @returns An object containing buy and sell objects.
  */
-// eslint-disable-next-line max-params -- Once bound to Client, getOrderbook only has 3 parameters.
+// eslint-disable-next-line max-params, complexity -- Once bound to Client, getOrderbook only has 3 parameters.
 async function getOrderbook(
   this: Client,
   takerPays: TakerAmount,
   takerGets: TakerAmount,
   options: {
     limit?: number
     ledger_index?: LedgerIndex
-    ledger_hash?: string
-    taker?: string
+    ledger_hash?: string | null
+    taker?: string | null
```

Comment:
Why this change?

Reply:
@mvadari The code was explicitly checking `undefined` and `null` which means `null` was missing from the type if that was indeed an expected value.

Reply:
I don't think this is necessary - I'd rather fix the check. It doesn't make sense to have a `null` param here.

Reply:
I'm not sure if that's the case, I'd rather leave it since it was already there and then if we want to fix it can do in another ticket --- unless you feel comfortable proposing a change